coffee_core/src/coffee.rs

```diff
@@ -195,10 +195,12 @@ impl CoffeeManager {
         if self.config.cln_config_path.is_none() {
             return Ok(());
         }
-        let root = self.config.cln_root.clone().unwrap();
+        let root = self.config.cln_root.clone();
+        let root = root.ok_or_else(|| error!("cln root path not found."))?;
         let rpc = Client::new(format!("{root}/{}/lightning-rpc", self.config.network));
         self.rpc = Some(rpc);
-        let path = self.config.cln_config_path.clone().unwrap();
+        let path = self.config.cln_config_path.clone();
+        let path = path.ok_or_else(|| error!("cln config path not found."))?;
```

Comment:
what about the path is `Some` but do not exist?

Reply:
The next line is 
```rust
let mut file = CLNConf::new(path.clone(), true);
```
where we set `create_if_missing` to `true` 
This is what happens when the user first runs coffee

Reply:
Mh I do not think this is right, if the `path` do not exist this code do not make sense because `CLNConf::new(path.clone(), true);` is not able to find the path where to create the file.

Please test the user case that I described https://github.com/coffee-tools/coffee/issues/209

Reply:
You are right.
This line will handle the case that the root path exists but the configuration file doesn't.
However, it will fail if the root path doesn't exist.

Reply:
Added another check for this case.

Reply:
>This line will handle the case that the root path exists but the configuration file doesn't.

This is done already by CLNConf

Reply:
I am not sure `CLNConf` handles this case.
Could you give it another try?
To reproduce:
1- delete ` ~/.lightning/bitcoin` directory
2- try running coffee commands (eg. `coffee remote list`)
You will get the same error mentioned in the original issue

What `CLNConf` handles is the case when ` ~/.lightning/bitcoin/config` is missing but ` ~/.lightning/bitcoin` exists.

Reply:
>I am not sure CLNConf handles this case.

yes is does, you need to do `CLNConf::new(path.clone(), false);` if the file do not exist it return an error

>What CLNConf handles is the case when  ~/.lightning/bitcoin/config is missing but  ~/.lightning/bitcoin exists.

Correct, you should check if the `path` exist